tinygrad/tensor.py

```diff
@@ -860,6 +859,15 @@ def softplus(self, beta=1): return (1/beta) * (1 + (self*beta).exp()).log()
   def softsign(self): return self / (1 + self.abs())
 
   # ***** broadcasted elementwise mlops *****
+  @staticmethod
+  def _broadcast_tensors(*ts:Tensor, shape:Tuple[sint, ...]=()):
+    # TODO: what if shape is smaller than tensor shapes...
```

Comment:
`ValueError`?

also fyi i will use this for setitem too to broadcast assign input into the assigned area

Reply:
hehe I saw the TODO in your PR
I'll try to get this done quick so I don't block you.